backend/testDBVersion

```diff
@@ -1 +1 @@
-896c6802eac730e95176c76a7b6dcf8da31ec69b
+0740cc90d73505896191d1974380de46b6bf1162
```

Comment:
should that be included ?

Reply:
yes, had to do a db migration of the test database. Thsi is why this the hash commit has been updated.